project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_forms.scss

```diff
@@ -93,6 +93,81 @@
   text-transform: uppercase;
 }
 
+// ========================================================================== //
+// Event Form
+// ========================================================================== //
+.node-form {
+  .control-label {
+    clear: both;
+    display: block;
+  }
+
+  .form-control:not(textarea) {
+    color: $color-gray-dark;
+    font-style: italic;
+    background-color: #EBEAE4;
+    border: 0;
+    border-radius: 20px;
+
+    &:focus {
+      box-shadow: 0 0 8px rgba(0, 0, 0, .6);
+    }
+  }
+
+  .document_file {
+    display: none !important;
+  }
+
+  .hidden-input {
```

Comment:
It seems like the _id_ of the element can tell us the name of the field, can you tell what is it ?

![capacity4more_c4m_document_widget_tpl_php_at_2370cf7ffc52a45eb09b084ea8ad3e00696290d5_ _capacity4dev_capacity4more](https://cloud.githubusercontent.com/assets/5985683/17461949/d5e3402c-5ca4-11e6-9eea-0790d572caaa.gif)